### PR TITLE
Cargo orm security

### PR DESCRIPTION
**Pull request by ZeroNetAlpha**
Ore Redemption Machines have been moved to more publicly accessible areas on Box and Delta stations. All stations now have a Mineral Storage locked windoor in front of the ORM. Also moved the Cargo Protolathe into the mining dock on Box because Miners kept breaking into the cargo bay to grab shit on high pop rounds.

🆑 ORM Movement
tweak: Box Station ORM Moved to 2 tiles south of Cargo Office Door.
tweak: Delta Station ORM moved to the southernmost window of Cargo.
tweak: Box, Delta, Meta stations now have Windoors with Mineral Storage access blocking them.
/🆑

[why]: Because originally, people were moving the ORM to break into cargo. An earlier PR overcorrected and removed ORM access to anyone but cargo. Hopefully this works better as a middleground.

(This is just a copypasta of the original PR's CL and reasoning. Nothing has changed between the failed PR and this one besides actually learning how to mapmerge properly.

### CODE REVIEW

**Comment by ZeroNetAlpha** [comment]
222 and 192. Much more reasonable.

File: _maps/cit_map_files/BoxStation/BoxStation.dmm [hunk 1]
```diff
@@ -56896,11 +56901,7 @@
 /turf/open/floor/plasteel,
 /area/quartermaster/storage)
 "QoO" = (
-/obj/machinery/mineral/ore_redemption{
-	input_dir = 2;
-	output_dir = 1
-	},
-/obj/machinery/door/firedoor,
+/obj/effect/spawner/structure/window/reinforced,
 /turf/open/floor/plasteel,
```
Comment (deathride58) [inline _maps/cit_map_files/BoxStation/BoxStation.dmm]: >plasteel floors underneath windows

Please use a plating here

File: _maps/cit_map_files/Deltastation/DeltaStation2.dmm [hunk 2]
```diff
@@ -105034,12 +105034,7 @@
...
 /area/quartermaster/office)
 "ehK" = (
-/obj/machinery/mineral/ore_redemption{
-	input_dir = 4;
-	output_dir = 8
-	},
-/obj/machinery/door/firedoor,
-/obj/effect/turf_decal/stripes/box,
+/obj/effect/spawner/structure/window/reinforced,
 /turf/open/floor/plasteel/neutral,
```
Comment (deathride58) [inline _maps/cit_map_files/Deltastation/DeltaStation2.dmm]: Ditto

**Review by deathride58 — CHANGES_REQUESTED** [review]

**Review by deathride58 — APPROVED** [review]

**Comment by Poojawa** [comment]
TEST YOUR SHIT

**Comment by Poojawa** [comment]
the ORM is placed backwards.

https://gyazo.com/13f0d0d0769835abfc88f524b10648e7

**Pull request merged**